vue/build/check-versions.js

```diff
@@ -0,0 +1,54 @@
+'use strict'
```

Comment:
if only for build, and we don't specific version dependency, I think we don't need this